Add ColorfulCards asset pack

MicahBird opened the pull request:
**Type**
Cards

**Author**
Дмитрий Фомин (Dmitry Fomin)

**License**
CC0 1.0

**URL**
https://commons.wikimedia.org/wiki/File:UNO_cards_deck.svg"

Greetings! I'm a fan of this project and have had a great time playing it with friends! So I'd like to help out with this project by adding more cards! Although I would like to include them within the base TabletopClub asset pack, I couldn't figure out how to create sub-directories in the `cards/` folder and have them be recognized. I'd be curious to hear your thoughts!

elmodor commented:
This is pretty neat!
Sub-directories are not possible currently.

Not sure how drwhut wants to handle this - but I just wanted to add that you could add a savegame to the pack so it could be loaded via the "Games" option menu. You can find the info on how to do so here:
https://tabletop-club.readthedocs.io/en/latest/custom_assets/tutorials/add_save_files.html

Guess there's not much to set up to do with UNO, but I think it would be nice if it would appear in the Games menu :smile:

MicahBird commented:
Save game added! I also updated the card stack to be 108 cards to fit more with the original game. I also changed descriptions from "Colorful Numbered Cards" to be "Cards for playing the game 'One'". As "Uno" is unfortunately trademarked, I found it more fitting to use the word "One" to allude to the popular game!

drwhut commented:
Hey, thank you so much for your PR! And thank you for the kind words, I'm glad you enjoyed playing the game :)

Unfortunately, at this time I am not planning to add anything to the default asset pack, or bundle anything with the game, that is not in the public domain. This is mentioned in [the FAQ in the docs](https://tabletop-club.readthedocs.io/en/latest/general/about.html#can-you-add-my-favourite-board-game-to-tabletop-club), and there's also a list of public domain games that I plan to add to the default pack here: #28. The short story as to why is for legal safety - even if the artwork is original and uses an open license, the original game still has things like trademarks, and potentially patents as well. A game like UNO is especially concerning, since the company who hold the rights currently are making a lot of money off of their video game adaptation. So they may very well just knock on the door and ask to remove it, and that's a fight I do not have the means to fight at the moment.

However, that only means I won't bundle it with official builds of the game - you are still free to share it amongst your friends, and eventually there will be some sort of copyright-friendly CDN (whether it's the Steam Workshop, or another system entirely) that you could upload it to :)
